spec/integration/manifest.spec.yml

```
@@ -221,9 +221,7 @@ spec:
             type: string
             oneOf:
               - type: string
-                pattern: '^http(s)?://'
-              - type: string
-                pattern: '^kbn:'
+                pattern: '^(http(s)?://|kbn:)'
```

Should we add then the slash also here?
```suggestion
                pattern: '^(http(s)?://|kbn:/)'
```